src/components/v5/common/ActionSidebar/partials/AmountField/types.ts

```diff
@@ -6,4 +6,5 @@ export interface AmountFieldProps {
   name: string;
   amount?: string;
   defaultToken?: string;
+  tokenAddressDisabled?: boolean;
```

Comment:
Instead I would just add an optional `tokenAddress` prop then if it's set I would just render this part
![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/af5eb8a2-992d-4321-8dd8-f107ad0657ba)
but without button.
Then in SimplePaymentForm in transform all transaction would just use token from `amount` field.